.github/workflows/docker-verify.yaml

```diff
@@ -0,0 +1,57 @@
+name: Docker verify process
+
+on:
+  pull_request:
```

Comment:
Maybe do on push to folder, so we do not trigger it every time? 
Something like:

```yaml
on:
  push:
    paths:
      - 'infrastructure/docker/**'
```

Or we have a requirements to run it on each PR regardless?

Reply:
Nope, I think it's a good idea. I'm running it in this PR every time to test that it works but before merge I think you are totally right 👍

Reply:
Longer-term, this might get faded out if we move the dockerfiles as discussed in https://github.com/Qiskit-Extensions/quantum-serverless/discussions/625 ...

Reply:
Included in: [04c616f](https://github.com/Qiskit-Extensions/quantum-serverless/pull/689/commits/04c616fd89ec0484f18210db8f0c40151a351735)